pkg/meta/sql.go

```diff
@@ -322,8 +322,38 @@ func (m *dbMeta) Init(format *Format, force bool) error {
 		if err != nil {
 			return fmt.Errorf("json: %s", err)
 		}
+		if old.DirStats && !format.DirStats {
+			// remove dir stats
+			_, err = m.db.Where("TRUE").Delete(new(dirStats))
+			if err != nil {
+				return errors.Wrap(err, "drop table dirStats")
+			}
+		}
+		if !old.DirStats && format.DirStats {
```

Comment:
this could be slow, and it should not block `Init()`.

The stats could be rebuild using `fsck`

Reply:
done